Test/TestDeltaMassPeak.cs

```diff
@@ -12,23 +12,23 @@ public class TestDeltaMassPeak
         [Test]
         public void TestDeltaMassPeakConstructor()
         {
-            Proteoform pf1 = new Proteoform("acession1");
```

Comment:
With type-checking or checking the ProteoformComparison type, I'm not sure this level of specificity is necessary here.